pkg/xds/experimental/client/option.go

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package client
+
+import (
+	"time"
+
+	"github.com/cilium/cilium/pkg/envoy"
+
+	"google.golang.org/grpc/codes"
+)
+
+type Options struct {
+	MinBackoff         time.Duration
```

Comment:
Suggest inner struct

```
type BackoffParams struct {
  Min, Max, Backoff time.Duration
}

type Options struct {
  RetryBackoff BackoffParams
  ...
}
```

Reply:
Done. This makes it much clearer. Thanks!